ecom-css/cart-grid-layout.css

```diff
@@ -0,0 +1,30 @@
+body {
+  height: 100vh;
+}
+
+.landing-container {
+  min-height: 100%;
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: auto repeat(3, 1fr) auto;
+  grid-template-areas:
+    "nav nav nav nav nav nav"
+    "contentarea contentarea contentarea contentarea contentarea contentarea"
+    "contentarea contentarea contentarea contentarea contentarea contentarea"
+    "contentarea contentarea contentarea contentarea contentarea contentarea"
+    "footerarea footerarea footerarea footerarea footerarea footerarea";
+}
```

Comment:
no need to repeat them if they are same. you can do this instead: 

"nav"
"contentarea"
"footerarea"